src/Handler/StreamHandler.php

```diff
@@ -152,8 +152,7 @@ private function createSink(StreamInterface $stream, array $options): StreamInte
             return $stream;
         }
 
-        $sink = $options['sink']
-            ?? \fopen('php://temp', 'r+');
+        $sink = $options['sink'] ?? Psr7\try_fopen('php://temp', 'r+');
```

Comment:
In what scenario can't you open `php://temp`?

Reply:
Need to use the new static methods.